### DISCUSSION TITLE
Is there a concise explanation of field arrays somewhere?

### DISCUSSION THREAD

**Discussion by darikg** [pyvista/pyvista#1499] [Q&A] [answered]
I understand point and cell arrays but I'm confused about field arrays. I haven't needed to use them before, but specifically I'm trying to do something with `vtkCollisionDetectionFilter`. It returns its results as a field array that (I think) is just an index vector into cells. How does vtk know that this index vector references cells and not points? Is there a quick demo somewhere on working with field arrays?

**Comment by tkoyama010** [comment, marked as answer]
As far as I know, we don't have a documentation about field array (https://github.com/pyvista/pyvista/issues/209 ). We will make demo using [CollisionDetection Example](https://kitware.github.io/vtk-examples/site/Python/Visualization/CollisionDetection/) about field.

**Reply by akaszynski** [reply]
FYI, working on this now.  It will be a fun demo.
